libraries/ms-helper/inc/can_uart.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include "can_hw.h"
+#include "status.h"
+#include "uart.h"
+
+// Generic CAN HW <-> UART protocol
+// Requires CAN HW, UART to be initialized
+
+// Process RX'd CAN message from UART
+struct CanUart;
+typedef void (*CanUartRxCb)(const struct CanUart *can_uart, uint32_t id, bool extended,
+                            const uint64_t *data, size_t dlc, void *context);
+
+typedef struct CanUart {
```

Comment:
Naming is really inconsistent in this struct. I personally prefer `CanUart` to `UARTPort` it is just more readable. @karlding and @tituschow  thoughts?

Reply:
True, I think it might be worth it to standardize on the former naming scheme. I can make a new ticket for it because it would bloat this PR. The issue with the latter is that this would become `CANUART`, which isn't very nice.

Thoughts @karlding?

Reply:
Sure